hack/generate-yamls.sh

```diff
@@ -107,66 +104,31 @@ cat "${SERVING_CORE_YAML}" > "${SERVING_YAML}"
-
-# Logs via ElasticSearch, Fluentd & Kibana
-ko resolve ${KO_YAML_FLAGS} -R \
-    -f third_party/config/monitoring/logging/elasticsearch \
-    -f config/monitoring/logging/elasticsearch | "${LABEL_YAML_CMD[@]}" > "${MONITORING_LOG_ELASTICSEARCH_YAML}"
-
-# Traces via Zipkin when ElasticSearch is installed
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/tracing/zipkin | "${LABEL_YAML_CMD[@]}" > "${MONITORING_TRACE_ZIPKIN_YAML}"
-
-# Traces via Zipkin in Memory when ElasticSearch is not installed
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/tracing/zipkin-in-mem | "${LABEL_YAML_CMD[@]}" > "${MONITORING_TRACE_ZIPKIN_IN_MEM_YAML}"
-
 echo "Building Monitoring & Logging"
 
-# By putting the list of files used to create the monitoring.yaml
-# people can choose to exclude certain ones via 'grep' but still keep in-sync
-# with the complete list if things change in the future
-echo "${MONITORING_LOG_ELASTICSEARCH_YAML}" >  "${MONITORING_FILES}"
-echo "${MONITORING_METRIC_PROMETHEUS_YAML}" >> "${MONITORING_FILES}"
-echo "${MONITORING_TRACE_ZIPKIN_YAML}"      >> "${MONITORING_FILES}"
-
-# Generate the core monitoring file - basically just the namespace
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/100-namespace.yaml \
-    | "${LABEL_YAML_CMD[@]}" > "${MONITORING_CORE_YAML}"
+ko resolve ${KO_YAML_FLAGS} -R \
+    -f config/monitoring/tracing \
+    -f config/monitoring/logging \
+    -f config/monitoring/metrics \
+    -f config/monitoring/third_party \
+    | "${LABEL_YAML_CMD[@]}" > "${MONITORING_FILES}"
 
 # Use ko to concatenate them all together.
 ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/100-namespace.yaml \
     $(sed "s/^/-f /" < "${MONITORING_FILES}") \
     | "${LABEL_YAML_CMD[@]}" > "${MONITORING_YAML}"
 
-# Traces via Jaeger when ElasticSearch is installed
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/tracing/jaeger/elasticsearch -f config/monitoring/tracing/jaeger/105-zipkin-service.yaml | "${LABEL_YAML_CMD[@]}" > "${MONITORING_TRACE_JAEGER_YAML}"
-
-# Traces via Jaeger in Memory when ElasticSearch is not installed
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/tracing/jaeger/memory -f config/monitoring/tracing/jaeger/105-zipkin-service.yaml | "${LABEL_YAML_CMD[@]}" > "${MONITORING_TRACE_JAEGER_IN_MEM_YAML}"
-
 echo "All manifests generated"
 
 # List generated YAML files, with serving.yaml first.
 
 cat << EOF > ${YAML_LIST_FILE}
 ${SERVING_YAML}
-${SERVING_CORE_YAML}
+${SERVING_FULL_YAML}
 ${SERVING_DEFAULT_DOMAIN_YAML}
 ${SERVING_STORAGE_VERSION_MIGRATE_YAML}
 ${SERVING_HPA_YAML}
 ${SERVING_CRD_YAML}
 ${SERVING_CERT_MANAGER_YAML}
 ${SERVING_NSCERT_YAML}
-${MONITORING_FILES}
-${MONITORING_YAML}
-${MONITORING_CORE_YAML}
-${MONITORING_METRIC_PROMETHEUS_YAML}
-${MONITORING_TRACE_ZIPKIN_YAML}
-${MONITORING_TRACE_ZIPKIN_IN_MEM_YAML}
-${MONITORING_TRACE_JAEGER_YAML}
-${MONITORING_TRACE_JAEGER_IN_MEM_YAML}
-${MONITORING_LOG_ELASTICSEARCH_YAML}
```

Comment:
This drops monitoring from the release, which I think needs further discussion.

Reply:
Ah, I see. Yeah, I should not drop it...